{{cookiecutter.repo_name}}/gulpfile.js/lib/postCssProcessors.js

```diff
@@ -13,6 +13,51 @@ module.exports = [
         gutter = gutter ? `${gutter}px` : 'var(--Grid_Gutter)'
 
         return `calc((${columns} / ${totalColumns} * 100%) - (${gutter} * 2) * (${totalColumns} - ${columns}) / ${totalColumns})`
+      },
+      smooth_gradient: function (direction, startingColor, easingFunction, finalColor = 'transparent', stops = 5, colorMode = '') {
```

Comment:
For consistency with how functions are always named in CSS, can this be called `smooth-gradient` rather than `smooth_gradient`? I hope postcss-functions allows this.

Reply:
(That should be "things that sort of look like function calls", like `linear-gradient`.)

Reply:
Nope. It can't have a `-` in it since it's a javascript function and it can't be camelCase since PostCSS didn't like that in the css files

Reply:
It is annoying I know

Reply:
What happens if you use a quoted string as the object key?